src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, value) {
+      operation(value);
+
+      return this;
+    },
+
+    add(value) {
+      calculator.result += value;
+
+      return this;
```

Comment:
those methods don't have to return `this`, you will pass your method to `operate` method and then invoke your operation with value, so only operate method has to return `this`